LiveSplit.AutoSplitters.xml

```diff
@@ -26098,6 +26098,28 @@
         <Type>Script</Type>
         <Description>Autosplitter/Load Removal available (by Jaffra)</Description>
     </AutoSplitter>
+    <AutoSplitter>
+        <Games>
+            <Game>Hack and Slash Fury</Game>
+        </Games>
+        <URLs>
+            <URL>https://github.com/just-ero/asl-help/raw/76750096863b03e7ebfdf748aef6d5d9464176a4/lib/asl-help</URL>
+            <URL>https://raw.githubusercontent.com/HandsomeStrife/speedrun-splits-and-autospltters/refs/heads/main/Hack%20and%20Slash%20Fury/hasf-autosplitter.asl</URL>
```

Comment:
The ASL file must be first.

Why don't you use the most recent commit of asl-help?